Encountify/Views/MapPage.xaml.cs

```diff
@@ -1,43 +1,69 @@
+using Encountify.Models;
+using Encountify.Services;
+using Encountify.ViewModels;
+using System;
+using System.Threading;
+using System.Diagnostics;
+using System.Collections.Generic;
 using Xamarin.Forms;
 using Xamarin.Forms.Xaml;
 using Xamarin.Forms.GoogleMaps;
-using Plugin.Geolocator;
-using System.Diagnostics;
-using System;
-using Encountify.Services;
-using System.Collections.Generic;
-using Encountify.Models;
-using Encountify.ViewModels;
+using Locations = Xamarin.Essentials.Location;
+using Geolocation = Xamarin.Essentials.Geolocation;
+using DistanceUnits = Xamarin.Essentials.DistanceUnits;
+using GeolocationRequest = Xamarin.Essentials.GeolocationRequest;
+using GeolocationAccuracy = Xamarin.Essentials.GeolocationAccuracy;
```

Comment:
This is a very nice way to simplify code (no need to change anything, just commenting to remember the place where it is)